src/foam/u2/wizard/common/SuccessWizardletView.js

```diff
@@ -0,0 +1,50 @@
+foam.CLASS({
+  package: 'foam.u2.wizard.common',
+  name: 'SuccessWizardletView',
+  extends: 'foam.u2.View',
+
+  css: `
+    ^ {
+      text-align: center;
+      display: flex;
+      flex-direction: column;
+      justify-content: center;
+      align-items: center;
+
+      height: 100%;
+    }
+    ^image {
+      width: 120pt;
+    }
+  `,
+
+  properties: [
+    {
+      class: 'String',
+      name: 'message',
+      value: 'Success!'
+    },
+    {
+      class: 'Image',
+      name: 'image',
+      view: 'foam.u2.view.ImageView',
+      // value: '/images/checkmark-small-green.svg'
+      value: '/images/checkmark-outline-green.svg'
+    }
+  ],
+
+  methods: [
+    function render() {
+      this
+        .addClass(this.myClass())
+        .startContext({ data: this })
+          .start(this.IMAGE)
+            .addClass(this.myClass('image'))
+          .end()
+        .endContext()
+        .start('h2')
```

Comment:
Should use `.addClass(‘h200’)` instead of the h2 tag. Allows us more control over styling and keeps it consistent across browsers

Reply:
Oh thanks, I was actually wondering about that but forgot to ask